pymc/smc/sample_smc.py

```diff
@@ -56,50 +56,53 @@ def sample_smc(
 
     Parameters
     ----------
-    draws: int
+    draws : int, default 2000
         The number of samples to draw from the posterior (i.e. last stage). And also the number of
         independent chains. Defaults to 2000.
-    kernel: SMC Kernel used. Defaults to pm.smc.IMH (Independent Metropolis Hastings)
-    start: dict, or array of dict
+    kernel : class, default `pymc.smc.smc.IMH`
+        SMC_Kernel used. Defaults to :class:`pymc.smc.smc.IMH` (Independent Metropolis Hastings)
```

Comment:
```suggestion
    kernel : SMC_kernel, optional
        SMC kernel used. Defaults to :class:`pymc.smc.smc.IMH` (Independent Metropolis Hastings)
```

The `SMC_Kernel` alias only works in the parameter type section.